src/module/Event/test/EventTest/Entity/EventLogTest.php

```diff
@@ -0,0 +1,146 @@
+<?php
+/**
+ * This file is part of Athene2.
+ *
+ * Copyright (c) 2013-2019 Serlo Education e.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @copyright Copyright (c) 2013-2019 Serlo Education e.V.
+ * @license   http://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/serlo-org/athene2 for the canonical source repository
+ */
+
+namespace EventTest\Entity;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Event\Entity\EventInterface;
+use Event\Entity\EventLog;
+use Event\Entity\EventLogInterface;
+use Event\Entity\EventParameterInterface;
+use User\Entity\UserInterface;
+use Uuid\Entity\UuidInterface;
+
+class EventLogTest extends \PHPUnit_Framework_TestCase
```

Comment:
@Knorrke First test for `::toJson()`. Feedback welcome :)

Reply:
Looks good to me 👍 Don't know if there's an easier way to mock this...

Reply:
Refactored it a little bit (see https://github.com/serlo/athene2/pull/853/commits/4ba6f1506e483b42370bdb7e23a9aa4471c72348), so it should be easier for the cases where you don't care much about the underlying object.